setup-kicad-library-utils/action.yml

```diff
@@ -0,0 +1,53 @@
+name: Set up KiCad Library Utils
+description: |
+  Installs KiCad Library Utils to the runner environment.
+
+  For more information, see:
+  https://gitlab.com/kicad/libraries/kicad-library-utils
+
+inputs:
+  version:
+    description: |
+      Git ref (branch, tag, or commit) of KiCad Library Utils to be installed
+    default: master
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Install dependencies
+      shell: bash
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y git python3 python3-pip
+
+    - name: Clone KiCad Library Utils
+      shell: bash
+      run: |
+        sudo git clone \
+          https://gitlab.com/kicad/libraries/kicad-library-utils.git \
+          /opt/kicad-library-utils
+        cd /opt/kicad-library-utils
+        sudo git checkout "${{ inputs.version }}"
```

Comment:
- `sudo` is unnecessary
- We don't need to `cd`, just perhaps use the `--branch` flag to checkout the right commit all at once?
- Use `--depth 1` because we are not interested in the other commits?